### DISCUSSION TITLE
Improvements on Pivot Tables.

### DISCUSSION THREAD

**Discussion by felipegranado** [apache/superset#29644] [Ideas]
Hello!

I wish Pivot Tables could have the options below:

1. Start the Chart with the aggregated data by default;

2. Have the option of having these groupings without having to have SUBTOTAL activated.

3. Allow placing an empty line between the groupings (smooth the visualization);

4. Having the possibility of changing the name of the column titles (dimensions) (it is locked with the name that comes in the Dataset).

Below is an illustrative image:

![image](https://github.com/user-attachments/assets/ef1ee323-199e-433f-bbad-cdbecc8289bf)

This way, we gain even more possibilities when creating our dashboards!

Thanks in advance!

**Comment by JoaoFabbri** [comment]
Hello!
Great points! 
It would be great if these adjustments were made in the superset, so that we, analysts, can always guarantee a more organized, clean and explicit look for the analyses. 🙌🏻
